terraform/modules/services/airflow/dags/uk/sat-dag.py

```diff
@@ -0,0 +1,103 @@
+"""Dag to download and process satellite data from EUMETSAT.
+
+Consists of two tasks made from the same ECS operator,
+one for RSS data and one for Odegree data.
+The 0degree data task only runs if the RSS data task fails.
+"""
+
+import os
+import datetime as dt
+from airflow.providers.amazon.aws.operators.ecs import EcsRunTaskOperator, EcsRegisterTaskDefinitionOperator
+from airflow.decorators import dag
+from airflow.utils.trigger_rule import TriggerRule
+
+from airflow.operators.latest_only import LatestOnlyOperator
+from utils.slack import slack_message_callback
+
+env = os.getenv("ENVIRONMENT", "development")
+
+default_dag_args = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "retries": 1,
+    "retry_delay": dt.timedelta(minutes=1),
+    "max_active_runs": 10,
+    "concurrency": 10,
+    "max_active_tasks": 10,
+    "execution_timeout": dt.timedelta(minutes=30),
+}
+default_task_args = {
+    "cluster": f"Nowcasting-{env}",
+    "task_definition": "satellite-consumer",
+    "launch_type": "FARGATE",
+    "task_concurrency": 1,
+    "network_configuration": {
+        "awsvpcConfiguration": {
+            "subnets": [os.getenv("ECS_SUBNET")],
+            "securityGroups": [os.getenv("ECS_SECURITY_GROUP")],
+            "assignPublicIp": "ENABLED",
+        },
+    },
+    "awslogs_group": "/aws/ecs/consumer/sat",
+    "awslogs_stream_prefix": "streaming/sat-consumer",
+    "awslogs_region": "eu-west-1",
+}
+
+satellite_error_message = (
+)
+
+@dag(
+    dag_id="uk-satellite-consumer",
+    description=__doc__,
+    schedule_interval="*/5 * * * *",
+    start_date=dt.datetime.now(tz=dt.timezone.utc) - dt.timedelta(hours=0.5),
+    catchup=False,
+    default_args=default_dag_args,
+)
+def sat_consumer_dag():
+
+    latest_only = LatestOnlyOperator(task_id="latest_only")
+    
+    sat_consumer_rss = EcsRunTaskOperator(
+        task_id="satellite-consumer-rss",
+        overrides={"containerOverrides": [{
+            "name": "satellite-consumer",
+            "environment": {
+                "SATCONS_SATELLITE": "rss",
+                "SATCONS_WORKDIR": "s3://nowcasting-sat-{env}/testdata",
```

Comment:
should this be in the main.tf ecs defiantely, and using the s3 bucket properties

Reply:
It could be, but I imagine it might want to be different for different instances of the consumer airflow task (e.g. india) so I left it in Airflow in order to be overwritten accordingly

Reply:
ah yes